book/src/terminology.md

```diff
@@ -18,13 +18,13 @@ The result of interpreting all programs on the ledger at a given [tick height](t
 
 A contiguous set of [entries](terminology.md#entry) on the ledger covered by a [vote](terminology.md#ledger-vote). A [leader](terminology.md#leader) produces at most one block per [slot](terminology.md#slot).
 
-## block height
+## blockhash
 
-The number of [blocks](terminology.md#block) beneath the current block. The first block after the [genesis block](terminology.md#genesis-block) has height zero.
+A preimage resistant [hash](terminology.md#hash) of the [ledger](terminology.md#ledger) at a given [block height](terminology.md#block-height). Taken from the last [entry id](terminology.md#entry-id) in the slot
 
-## block id
+## block height
 
-The [entry id](terminology.md#entry-id) of the last entry in a [block](terminology.md#block).
+The number of [blocks](terminology.md#block) beneath the current block. The first block after the [genesis block](terminology.md#genesis-block) has height zero.
```

Comment:
@jstarry, is that true? height zero?

Reply:
Yes, it's currently true, but very confusing. We should probably not be zero-indexing block height.. https://github.com/solana-labs/solana/issues/6779

Reply:
🤔 Must be some greedy diff aggregation going on here.  That's not a line that I touched